uaclient/entitlements/repo.py

```diff
@@ -192,24 +195,24 @@ def setup_apt_config(self) -> None:
         aptKey = directives.get('aptKey')
         if not aptKey:
             raise exceptions.UserFacingError(
-                "Ubuntu Advantage server provided no aptKey directive for %s."
-                % self.name
+                "Ubuntu Advantage server provided no aptKey directive for"
```

Comment:
What do you think about single quotes instead of double throughout? I'd like to avoid double-quotes as  as style choice unless we needed to embed single quotes within the string.

Reply:
If we want to standardise on string quoting (and bring it up in review), I think we should undo my (solo) decision to have black not normalise strings, and have it handle quotes too.

Reply:
nice thought. yes let's have a discussion about this tomorrow and see what opinions are on this policy.